hedera-node/test-clients/build.gradle.kts

```diff
@@ -13,8 +13,6 @@ mainModuleInfo {
     runtimeOnly("org.junit.platform.launcher")
 }
 
-testModuleInfo { runtimeOnly("org.junit.jupiter.api") }
```

Comment:
This not necessary - org.junit.jupiter.api is needed at compile time. So it is already accessible, otherwise compilation would not work.

Reply:
Probably unintentionally added by an IDE